src/MSBuild.UnitTests/Microsoft.Build.CommandLine.UnitTests.csproj

```
@@ -75,4 +75,22 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Snapshots\TerminalLogger_Tests.PrintRestore_Failed.Linux.verified.txt">
```

```suggestion
    <None Update="Snapshots\*.verified.txt">
```

To make it apply to all verified files with only one `<None Update` ?